### PR TITLE
Update redis.txt to 4.3.6 to fix severe issue

### PR DESCRIPTION
**Pull request by auvipy**

### CODE REVIEW

**Comment by auvipy** [comment]
should be updated based on https://github.com/redis/redis-py/issues/2665

File: requirements/extras/redis.txt [hunk 1]
```diff
@@ -1 +1 @@
-redis>=4.2.2,<4.4.0
+redis>=4.3.6,<4.4.0
```
Comment (auvipy) [inline requirements/extras/redis.txt:1]:
```suggestion
redis>=4.3.6,<4.6.0
```

**Comment by auvipy** [comment]
>               assert connparams['health_check_interval'] == 25
E               KeyError: 'health_check_interval'

t/unit/transport/test_redis.py:867: KeyError

File: requirements/extras/redis.txt [hunk 2]
```diff
@@ -1 +1 @@
-redis>=4.2.2,<4.4.0
+redis>=4.3.6,<4.6.0
```
Comment (chayim) [inline requirements/extras/redis.txt:1]: I think you now want 4.3.7.  GitHub pipelines seem plugged - but it should get there as soon as things wake up.

Comment (auvipy) [reply requirements/extras/redis.txt:1]: yes but I haven't seen any release for that yet https://github.com/redis/redis-py/releases

**Comment by auvipy** [comment]
also we need some suggestion to fix the failing test while upgrading from 4.3 to 4.4.x+



              assert connparams['health_check_interval'] == 25

E KeyError: 'health_check_interval'

t/unit/transport/test_redis.py:867: KeyError

File: requirements/extras/redis.txt (hunk 2 above)
Comment (auvipy) [inline requirements/extras/redis.txt:1]:
```suggestion
redis>=4.3.6,<4.4.0
```

**Pull request merged**